test/jdk/java/nio/file/attribute/BasicFileAttributeView/CreationTime.java

```diff
@@ -108,6 +108,10 @@ static void test(Path top) throws IOException {
         } else if (Platform.isLinux()) {
             // Creation time read depends on statx system call support
             supportsCreationTimeRead = Linker.nativeLinker().defaultLookup().find("statx").isPresent();
+            // Linux system doesn't support birth time on tmpfs filesystem for now
+            if( Files.getFileStore(file).type().contentEquals("tmpfs") ) {
```

Comment:
`if( ` should be `if (` but I would suggest instead:

```
// Creation time read depends on statx system call support and on the file
// system storing the birth time. The tmpfs file system type does not store
// the birth time.
boolean statxIsPresent = Linker.nativeLinker().defaultLookup().find("statx").isPresent();
if (statxIsPresent && !Files.getFileStore(file).type().contentEquals("tmpfs")) {
    supportsCreationTimeRead = true;
}
```

Reply:
Thanks for your review and suggestion. The code has been updated accroding your suggestion.